### DISCUSSION TITLE
Components Overview for Flynt

### DISCUSSION THREAD

**Discussion by timohubois** [flyntwp/flynt#486] [Show and tell]
Want to get an overview where components are currently used? 

There is a plugin that can do this for you:
GitHub: https://github.com/timohubois/components-overview-flynt
WordPress Plugin Repo: https://wordpress.org/plugins/components-overview-flynt

All feedback is welcome and everyone can contribute 🚀

**Comment by philipphaunstetter** [comment]
Thank you <3!
